app/controllers/core_files_controller.rb

```diff
@@ -93,7 +93,12 @@ def update
   end
 
   def view_package_html
-    @core_file = CoreFile.find_by_did(params[:did])
+    # the :did attribute has been removed since it's not part of the new version of the app
+    # but it may need to be added again to store legacy did's? not sure yet
+    # decide what to do with did values that are associated with core files in the current prod database as those files are imported into the new database
+    core_file_id = params[:id] || params[:did]
+
+    @core_file = CoreFile.find_by_id(core_file_id)
```

Comment:
This seems like an important piece to resolve either before merging this PR, or immediately after as a separate issue once this PR is merged.

Reply:
I created issue #99 for this. My initial thought is that the import process we plan to use for getting the production app db data into the v2 app db will also guide this task of refactoring methods related to identifying core files for requests to the controller overall. 

i.e., we can store both the v2 app db generated IDs and URLs for core files and the production db DIDs and URLs for core files by blending the v2 app core file db table structure with the production core file db table structure (dropping whatever fields we know we won't need to persist to v2) when creating the sql script to generate the db dump file to import. 

once the blended table exists in the v2 app db, it will be simpler to refactor this logic, though the logic itself will be fairly complex as you stated.